ceno_zkvm/benches/fibonacci.rs

```diff
@@ -0,0 +1,87 @@
+use std::{
+    fs,
+    path::PathBuf,
+    time::{Duration, Instant},
+};
+
+use ceno_emul::{CENO_PLATFORM, Platform, Program, WORD_SIZE};
+use ceno_zkvm::{
+    self,
+    e2e::{run_e2e_gen_witness, run_e2e_proof},
+};
+use criterion::*;
+
+use goldilocks::GoldilocksExt2;
+use mpcs::BasefoldDefault;
+
+criterion_group! {
+  name = fibonacci;
+  config = Criterion::default().warm_up_time(Duration::from_millis(20000));
+  targets = bench_e2e
+}
+
+criterion_main!(fibonacci);
+
+const NUM_SAMPLES: usize = 10;
+
+fn bench_e2e(c: &mut Criterion) {
+    type Pcs = BasefoldDefault<E>;
+    let mut file_path = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
+    file_path.push("examples/fibonacci.elf");
```

Comment:
Please see how `test_elf.rs` builds and reads in the examples.

Reply:
Build and read elf is outside of the scope. there are other ongoing task for build fibonacci elf.